### DISCUSSION TITLE
Slack inactive?

### DISCUSSION THREAD

**Discussion by rick0403** [saleweaver/python-amazon-sp-api#45] [Q&A]
The Slack links appears not to work. Is there anything active re this package going on there, or is it all here on Git?

**Comment by saleweaver** [comment]
Hi, I've tried slack but I don't think there are enough people on the repo atm. For now it's all here on git.

**Reply by rick0403** [reply]
Thanks its kinda a dumb question but it helps to be sure where to look and where not to bother and not fear missing the answer in an obvious place
